test/transaction.js

```diff
@@ -196,16 +196,14 @@ describe('Transaction', function() {
         b: 'b',
       };
 
-      const expectedError = {
-        code: 4,
-        message: 'Deadline for Transaction exceeded.',
-        a: 'a',
-        b: 'b',
-      };
-
       const formattedError = Transaction.createDeadlineError_(originalError);
 
-      assert.deepStrictEqual(expectedError, formattedError);
+      assert.strictEqual(formattedError.code, 4);
```

Comment:
Can we check that we are returning a "common.util.ApiError"?

Reply:
Sure. that seems like a reasonable additional check.